README.md

```diff
@@ -116,7 +167,8 @@ $ cmcli wifi connect --ssid AnAccessPoint
 A password will be prompted for if one is required. `cmcli` will exit when the access point has been
 connected to or an error has occurred, in which case an error message should have been printed.
 
-# License and Copyright
+License and Copyright
+=====================
 
 Copyright © 2019 Luxoft Sweden AB
```

Comment:
Maybe we should add 2020 ... Do not remember the details of when or how, but I have a vague notion that this is something to consider.

Reply:
I do not want to flog a dead horse so I will just leave this alone. ;)